docs/reference/ml/apis/evaluateresources.asciidoc

```diff
@@ -0,0 +1,50 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ml-evaluate-dfanalytics-resources]]
+=== Evaluate {dfanalytics} resources
+
+The evaluation object defines the type of evaluation you want to perform. An 
+evaluation configuration object has the following properties:
+
+`binary_soft_classification`::
+  (object) Configures a binary soft classification type evaluation.
+  
+  
+[float]
+[[binary-sc-resources]]
+==== Binary soft classification resources
+
+A binary soft classification object has the following properties:
```

Comment:
Here we should explain when this evaluation is suitable. Something along the lines of:

Binary soft classification is used in order to evaluate the results of an analysis which outputs the probability that each data frame row is of a certain class. For example, in the context of outlier detection, the analysis outputs the probability that each row is an outlier.